Sources/UI/Chat/Chat View Controller/ChatViewController.swift

```diff
@@ -308,6 +308,19 @@ open class ChatViewController: ViewController, UITableViewDataSource, UITableVie
         defaultActionsContextMenu(from: cell, for: message, locationInView: locationInView)
     }
     
+    /// Creates a chat view controller for the message being replied to. Override this to change style and other properties of the thread's view controller.
```

Comment:
This should be on a different line, in discussion:
```swift
/// Creates a chat view ....
/// 
/// Override this to style...
```

Reply:
Done! Thanks